portugues/colabora/comdiferencas/diff/pull/pull.md

```diff
@@ -1 +1,5 @@
-Então..
\ No newline at end of file
+Então espero o desenvolvedor responder meu pull request com seu feedback :).
+
+[Não deu certo? Ele não gostou? Volte ao trabalho](diff/diff.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.